src/components/footnote/footnote.scss

```diff
@@ -0,0 +1,46 @@
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/focus';
+@use '../../mixins/ms';
+
+$footnote-offset: ms.step(-2);
```

Comment:
I wouldn't mind if this was a tad bit larger to give the background a bit of breathing room at the top. But, I'll defer to you and the @cloudfour/design team. 😉 

<img width="397" alt="Screen Shot 2021-09-02 at 10 28 19 AM" src="https://user-images.githubusercontent.com/459757/131889920-898f444f-bdb0-414e-a10f-38bd6e8fa76a.png">

Reply:
I think the default browser behavior is to hug the top of the viewport pretty closely. So while I agree from a visual design standpoint, from a behavior standpoint I wonder if the more modest adjustment to account for padding would match the user's expectation more closely? 🤔 

I'm not 100% certain, though.

Reply:
That's fair. I don't feel strongly about this so no changes are required. 🙂

Reply:
That's a good question... I don't have super strong feelings either way. I can experiment and see what feels natural

Reply:
I added a tad of additional spacing and it felt pretty natural to me

https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/ec7dcd347c32f5e861850b482733a257ce6c4044